components/dashboard/challenge-card.tsx

```
@@ -19,38 +20,141 @@ export function ChallengeCard({
+		<div className="min-h-screen bg-gray-50 dark:bg-gray-900 p-4">
+			<div className="max-w-6xl mx-auto">
+				<h1 className="text-xl sm:text-2xl lg:text-3xl font-bold text-gray-900 dark:text-white mb-6 sm:mb-8">
+					Challenge Cards - Responsive Demo
+				</h1>
+				
+				{/* Grid layout - responsive columns */}
+				<div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-4 sm:gap-6">
+					{challenges.map((challenge, index) => (
+						<ChallengeCard
+							key={index}
+							title={challenge.title}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)